foreman/data_refinery_foreman/foreman/management/commands/correct_affy_cdfs.py

```diff
@@ -92,7 +92,13 @@ def handle(self, *args, **options):
                     # GEOparse downloads files here and never cleans them up! Grrrr!
                     download_path = GEO_TEMP_DIR + experiment.accession_code + '_family.soft.gz'
                     # It's not a directory, but ignore_errors is useful.
-                    shutil.rmtree(download_path, ignore_errors=True)
+                    try:
+                        os.remove(download_path)
+                    except:
+                        # Don't anything interrupt this, like say,
+                        # GEOParse downloading a directory instead of
+                        # a file...
+                        logger.exception("Failed to delete an archive.")
```

Comment:
Does this include the exception that was raised?

Reply:
Yup, that's what `logger.exception` does.